src/main/java/frc/robot/TunableConstant.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class TunableConstant {
+  private String key;
+  private double defaultValue;
+  private double previousValue;
+  
+  public TunableConstant(String key, double defaultValue) {
+    this.key = key;
+    this.defaultValue = defaultValue;
+    SmartDashboard.putNumber(key, SmartDashboard.getNumber(key, defaultValue));
+  }
+
+  public double get() {
```

Comment:
It took me a while to figure out how this class works. Maybe renaming `get()` to `update()` would clarify the API.

Reply:
I like get for the public API, but I moved the update logic into a private update method. Hope that's clearer.